CMakeLists.txt

```diff
@@ -94,7 +95,13 @@ else(MSVC)
   endif()
   check_cxx_compiler_flag("-std=c++11"   SUPPORT_CXX11)
   check_cxx_compiler_flag("-std=c++0x"   SUPPORT_CXX0X)
-  check_cxx_compiler_flag("-msse2"       SUPPORT_MSSE2)
+  # For cross compilation, we can't rely on the compiler which accepts the flag, but mshadow will
+  # add platform specific includes not available in other arches
+  if(USE_SSE STREQUAL "AUTO")
+    check_cxx_compiler_flag("-msse2"       SUPPORT_MSSE2)
```

Comment:
Why would you ever turn it off on a platform that supported it?

Reply:
When we cross compile, the compiler supports the option, but the target platform actually does not support it. This means that the cmake auto detects it wrong.

Reply:
ok

Reply:
nit: the spacing before SUPPORT_MSSE2 becomes kind of pointless if the check_cxx_compiler_flag(...) line isn;t still aligned with the rest of them